app/src/main/res/values/color_defs.xml

```diff
@@ -0,0 +1,158 @@
+  <color name="chapterCardGreyBorder">#666666</color>
+  <color name="divider">#61000000</color>
+  <color name="toolbarDropShadow">#3D000000</color>
+  <color name="green_shade">#44A85A</color>
+  <color name="green_shade_80">#CC44A85A</color>
+  <color name="light_green">#D9F3E7</color>
+  <!-- QUESTION COMPONENT -->
+  <color name="question_player_background">#F0FFFF</color>
+  <color name="question_player_progress_bar_gradient_start">#00E4F2F1</color>
+  <color name="question_player_progress_bar_gradient_center">#E4F2F1</color>
+  <color name="question_player_progress_bar_gradient_end">#E4F2F1</color>
+  <!-- AVATAR BACKGROUND COLORS -->
+  <color name="avatar_background_1">#E65C5C</color>
+  <color name="avatar_background_2">#E6935C</color>
+  <color name="avatar_background_3">#FFDB4E</color>
+  <color name="avatar_background_4">#CAE65C</color>
+  <color name="avatar_background_5">#93E65C</color>
+  <color name="avatar_background_6">#5CE6CA</color>
+  <color name="avatar_background_7">#5CCAE6</color>
+  <color name="avatar_background_8">#5C93E6</color>
+  <color name="avatar_background_9">#5C5CE6</color>
+  <color name="avatar_background_10">#935CE6</color>
+  <color name="avatar_background_11">#E65CCF</color>
+  <color name="avatar_background_12">#E65C97</color>
+  <color name="avatar_background_13">#CC2929</color>
+  <color name="avatar_background_14">#CC6A29</color>
+  <color name="avatar_background_15">#E6C12E</color>
+  <color name="avatar_background_16">#ABCC29</color>
+  <color name="avatar_background_17">#6ACC29</color>
+  <color name="avatar_background_18">#29CCAB</color>
+  <color name="avatar_background_19">#29ABCC</color>
+  <color name="avatar_background_20">#296ACC</color>
+  <color name="avatar_background_21">#6A29CC</color>
+  <color name="avatar_background_22">#AB29CC</color>
+  <color name="avatar_background_23">#CC29B1</color>
+  <color name="avatar_background_24">#CC2970</color>
+  <!-- ReadingTextSizeActivity -->
+  <color name="seekbarDivider">#707070</color>
+  <color name="topicThumbnailBackground">#C6DCDA</color>
+  <color name="topicTitleBackground">#2F6687</color>
+  <!-- STATUS BAR COLORS -->
+  <color name="profileStatusBar">#000000</color>
+  <color name="hintFullDialogStatusBar">#00236E</color>
+  <color name="slideDrawerOpenStatusBar">#001F2A</color>
+  <color name="walkthroughStatusBar">#CCCCCC</color>
+  <color name="onboarding1StatusBar">#00847A</color>
+  <color name="onboarding2StatusBar">#8B2A15</color>
+  <color name="onboarding3StatusBar">#AD8500</color>
+  <color name="onboarding4StatusBar">#3B5893</color>
+  <!-- ICON COLORS -->
+  <color name="mergeIconEnabled">#FF000000</color>
+  <color name="mergeIconDisabled">#999999</color>
+  <!-- HIGHLIGHTED COLORS -->
+  <color name="adminHighlighted">#A5D3EC</color>
+  <color name="comingSoonEndColor">#674172</color>
+  <color name="comingSoonStartColor">#7659B6</color>
+  <!-- DARK MODE COLORS -->
+  <color name="colorPrimary_DarkMode">#3bd1c4</color>
```

Comment:
In `color_defs` do not mentioned `dark` or `light` on the basis of mode in any color name.
Instead it should be like `white`, `black`, `gray_1`, `red`, `oppia_primary`, `oppia_light_green`  or `oppia_dark_green` etc.

Reply:
PTAL now.